.github/workflows/tests.yml

```diff
@@ -19,6 +19,10 @@ jobs:
             pip install poetry
             poetry install
 
+      - name: Install 3rd party from apt
+        run: |
+            sudo apt install unar
```

Comment:
This is already added in the `main` branch, so this modification will not be needed once this branch is rebased on top of the current `main`,

Reply:
Rebased on top of the current `main` so it's no longer there.